packages/sw-broadcast-cache-update/src/lib/responses-are-same.js

```diff
@@ -42,11 +43,30 @@ import ErrorFactory from './error-factory';
  */
 function responsesAreSame({first, second, headersToCheck}={}) {
   if (!(first instanceof Response &&
-        second instanceof Response &&
-        headersToCheck instanceof Array)) {
+    second instanceof Response &&
+    headersToCheck instanceof Array)) {
     throw ErrorFactory.createError('responses-are-same-parameters-required');
   }
 
+  const atLeastOneHeaderAvailable = headersToCheck.some((header) => {
+    return first.headers.has(header) && second.headers.has(header);
+  });
+  if (!atLeastOneHeaderAvailable) {
+    logHelper.log({
+      message: `Unable to determine whether the response has been updated
+        because none of the headers that would be checked are present.`,
+      data: {
+        'First Response': first,
+        'Second Response': second,
+        'Headers To Check': JSON.stringify(headersToCheck),
```

Comment:
I've found that there's some point (after the variable's gone out of scope in the main thread?) after which objects that are logged to the JS console are no longer "live", and arrays, for example, won't properly display their contents if you try to expand them. So logging the string representations of objects seems like the safest approach. We could theoretically move that logic into `logHelper.js`, but complex objects can't be `JSON.stringified()`.